### PR TITLE
Fix incorrect definitions in the czech republic calendar & Fix the drop feature so you now can ommit the date in a calendar definition

### PR DESCRIPTION
**Pull request by emagnier**
Some definition was incorrect in the Czech Republic calendar.

Strangely, Travis didn't say anything.

### CODE REVIEW

File: src/calendars/czechRepublic.ts [hunk 1]
```diff
@@ -202,6 +202,7 @@ const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> =
     },
     {
       key: 'lawrence_ruiz_and_companions_martyrs',
+      date: dayjs.utc(`${year}-9-28`),
```
Comment (tukusejssirs) [inline src/calendars/czechRepublic.ts]: AFAIK when dropping a celebration, romcal does not require defining a date, does it?

**Comment by tukusejssirs** [comment]
> Strangely, Travis didn't say anything.

Lately, Travis seems to miss some issues with testing romcal. Is it (still) reliable? :wink:

File: src/calendars/czechRepublic.ts (hunk 1 above)
Comment (emagnier) [reply src/calendars/czechRepublic.ts]: Actually it's required. But I agree that it makes no sense. 

I'll try to address that in this PR.

**Comment by emagnier** [comment]
> Lately, Travis seems to miss some issues with testing romcal. Is it (still) reliable? 😉

Yes, Travis doesn't seem to be very reliable. And I saw a few opened things in the issue's tracker about Travis, it might be a good time to review and address them (before releasing a final v2).

On my side I don't have too much time for that, maybe you @tukusejssirs or @pejulian could help on that part?

**Comment by emagnier** [comment]
I've added another commit. 

On a calendar definition: you now can omit the `date` property if you add the `drop` flag.

**Review by tukusejssirs — APPROVED** [review]

**Comment by tukusejssirs** [comment]
Mayyoy should update the PR title to reflect the `drop` change.

**Pull request merged**